roles/gitea/defaults/main.yaml

```
@@ -1,10 +1,14 @@
 ---
 gitea_namespace: gitea-system
-gitea_version: 3.1.3
+gitea_version: 4.0.0
 
 gitea_values: {}
 gitea_values_files: []
 
 gitea_port: "3000"
 
+gitea_use_istio: true
+gitea_create_virtualservice: "{{ gitea_use_istio }}"
```

nit: Consider keeping them separate? i.e.
```suggestion
gitea_use_istio: true
gitea_create_virtualservice: true
```

Unless we're saying it's impossible to create a virtualservice without istio (which is true since `gitea-vs.yaml.j2` is based on istio. :shrug:

---

It is impossible to create VirtualService without Istio :)